cmd/sync.js

```diff
@@ -136,9 +139,19 @@ const handler = function(argv) {
         process.exit(1)
       }
 
+      // converting comma separated list of labels to array
+      const labels = argv.addLabels ? new String(argv.addLabels).split(',') : []
```

Comment:
@trepel known issues is because of here should be a smarter check - I would say default value for addLabes undefined and then === check for undefined and false and 0 would be possible? ;-)

Reply:
I now I'm really enjoying being annoying 🤣